aws/resource_aws_volume_attachment.go

```diff
@@ -205,6 +204,10 @@ func resourceAwsVolumeAttachmentRead(d *schema.ResourceData, meta interface{}) e
 	return nil
 }
 
+func resourceAwsVolumeAttachmentUpdate(d *schema.ResourceData, meta interface{}) error {
+	return nil
```

Comment:
Can you add a log statement explaining that we are actually doing nothing along with a comment for the reason behind?